src/frontend/src/lib.rs

```diff
@@ -41,6 +41,7 @@ pub mod handler;
 pub mod observer;
 pub mod optimizer;
 pub mod planner;
+#[allow(unused)]
```

Comment:
Why not directly removed it?

Reply:
Some variants and fields still not used, so we can't remove it directly.